### DISCUSSION TITLE
Should we warn people about enabling the 'More links' addon?

### DISCUSSION THREAD

**Discussion by BoomerScratch** [ScratchAddons/ScratchAddons#916] [Discussion]
I have been inactive for a long time, sorry for that.

Anyways, clicking a malicious link could potentially send your login cookie to an attacker, allowing them to hack your account. Should we warn people about that?

**Comment by WorldLanguages** [comment]
> clicking a malicious link could potentially send your login cookie to an attacker

The web doesn't work like that. A navigation does not send third party cookies.

**Reply by pufferfish101007** [reply]
True, but bearing in mind a lot of scratch users are quite young, we probably should add a note reminding them not to go to links they don't recognise or that don't seem trustworthy.

**Reply by WorldLanguages** [reply]
@pufferfish101007 If they installed an extension then they kinda broke that rule already.

**Reply by pufferfish101007** [reply]
> @pufferfish101007 If they installed an extension then they kinda broke that rule already.

I can't argue with that one :man_shrugging:
Ok, my entire argument has been rendered reduntant, and I also realised that scratch automatically linkifies https://website.domain/page links that aren''t in `[url]`s, so this entire idea is redundant 😭

**Reply by BoomerScratch** [reply]
Oh, I thought it did :P
